UnitySDK/Assets/ML-Agents/Scripts/BroadcastHub.cs

```diff
@@ -11,50 +11,27 @@ namespace MLAgents
     [System.Serializable]
     public class BroadcastHub
     {
-        [SerializeField]
-        public List<Brain> broadcastingBrains = new List<Brain>();
         [FormerlySerializedAs("_brainsToControl")]
         [SerializeField]
-        private List<Brain> m_BrainsToControl = new List<Brain>();
+        public List<LearningBrain> brainsToControl = new List<LearningBrain>();
```

Comment:
Should we make this `[FormerlySerializedAs("broadcastingBrains"]`? or is that going to break loading old assets? What's the upgrade path for users?

Reply:
I updated all the scenes.

Reply:
Maybe I can just remove the FormerlySerializedAs. It was added automatically by my IDE